css/style.css

```diff
@@ -1,18 +1,334 @@
-.main {
-    display: flex;
+body {
+	box-sizing: content-box;
+	margin: 0;
+	background-color: #f9f9f9;
+	font-family: 'Roboto', sans-serif;
 }
-.navbar{
-    height: 60px;
+
+div, h4, h3, h1{
+	display: block;
+	margin: 0;
+	padding: 0;
+	background: transparent;
 }
-li {
-    list-style-type: none;
-    padding: 10px;
 
+#menu-icon {
+	display: flex;
+	align-items: center;
+	font-size: 1.5rem;
+	padding-left: 1rem;
+}
+
+
+i {
+	color: #606060;
+}
+
+
+
+.navbar {
+	background-color: white;
+	display: flex;
+	font-size: 30px;
+	justify-content: space-between;
+	position: fixed;
+	width: 100%;
+	z-index: 1;
+}
+
+.navbar-items {
+	display: flex;
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS, like `display: flex;`. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)


  👉 Here you have an explanation of how you can implement it:

  You can use multiple classes for any HTML element, so for example,

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
  and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```